src/Extensions/Abstractions/EventIds.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Event ids enum</summary>
```

Comment:
>summary [](start = 6, length = 7)

By convention, summaries are normally formatted as:

`/// <summary>
/// Event ids enum
/// </summary>`

Reply:
Applies throughout

---
In reply to: [362443319](https://github.com/microsoft/Omex/pull/153#discussion_r362443319) [](ancestors = 362443319)

Reply:
So it should be three comments signs in a single line? 
Could you please provide some links with information about it?

Reply:
The comment didn't format correctly it seems (although it looks fine in CodeFlow). See [https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/) as an example of the XML comment format.

---
In reply to: [362452169](https://github.com/microsoft/Omex/pull/153#discussion_r362452169) [](ancestors = 362452169)

Reply:
```
csharp
/// <summary>
/// Event ids enum
/// </summary>
```

Reply:
I don't see any requirement to have a summary in three separate lines, to me it feels like having more comments than code makes it harder to read.
Do you think it's necessary to make it into three-line everywhere?

Reply:
Personally, I find it a little confusing as is because I've never seen `<summary>` tags on a single line. As a result, my mind doesn't grasp it immediately as a summary when I'm scanning the code. Have you seen this done in other projects? I know we don't do it in OMEX, but perhaps this is standard practice for some teams

---
In reply to: [362761609](https://github.com/microsoft/Omex/pull/153#discussion_r362761609) [](ancestors = 362761609)

Reply:
I like it a bit more because then it takes less space (especially in enums), I think I've seen it in BlackWater code, but I might be mistaken. As you probably know I'm not a big fan of comments, especially the one that duplicate name, os if it's possible to include in standard I would prefer to allow a single line version.

Reply:
For now I've replaced all single line with multiline in this commit https://github.com/microsoft/Omex/pull/153/commits/050d53f6200815bd28c5eda59a66d951061a586b
if you would like to compare difference visually